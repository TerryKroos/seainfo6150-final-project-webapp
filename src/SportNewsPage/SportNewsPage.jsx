import React from 'react'
import NavBar from "../NavBar/NavBar";
import Footer from "../Footer/Footer";
import ArticleListOther from "../ArticleListOther/ArticleListOther.jsx";


const SportNewsPage = (props) => {
    return (
       <div >
        <NavBar/>
        <ArticleListOther articles={props.articles}/>
        <Footer/>
       </div>
    );
}

export default SportNewsPage;