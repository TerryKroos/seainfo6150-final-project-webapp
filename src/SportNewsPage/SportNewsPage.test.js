



import React from "react";
import SportNewsPage from "./SportNewsPage";
import { render } from "@testing-library/react";

describe("SportNewsPage tests", () => {
    it("renders correctly", () => {
      const { container } = render(<SportNewsPage articles={[]}/>);
      expect(container).toMatchSnapshot();
    });
  
  });