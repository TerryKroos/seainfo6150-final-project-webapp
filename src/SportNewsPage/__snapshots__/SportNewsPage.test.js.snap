// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SportNewsPage tests renders correctly 1`] = `
<div>
  <div>
    <nav>
      <ul>
        <li>
          The Sports Blog
        </li>
        <li>
          <a
            class="link"
            href="/"
          >
            HOME
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/nba"
          >
            NBA
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/nfl"
          >
            NFL
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/nhl"
          >
            NHL
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/mbl"
          >
            MBL
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/soccer"
          >
            SOCCER
          </a>
        </li>
      </ul>
    </nav>
    <div
      class="container"
    >
      <ul
        class="articles"
      />
    </div>
    <footer>
      <ul>
        <li>
          <a
            class="link"
            href="/about"
          >
            About
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/subscribe"
          >
            Subscribe
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/privacy"
          >
            Privacy
          </a>
        </li>
      </ul>
    </footer>
  </div>
</div>
`;
